### PR TITLE
Commander with cli-builder

### PR DESCRIPTION
**Pull request by maarteNNNN**
It's ready for testing. `verify` in transactions is not implemented yet. The `README` provides all available commands. Supports both interactive or per command.

Project is using the `cli-builder` that I made open-source but doesn't use additional npm modules other than the ones the project was already using.

Let me know what you think.

### CODE REVIEW

File: bin/cli.js [hunk 1]
```diff
@@ -1,15 +1,297 @@
...
-
-let command = argv._[0];
-
-// TODO: Implement
+const ldposClient = require('ldpos-client');
+const { CliInterface } = require('@maartennnn/cli-builder');
+const actions = require('../lib/actions');
+const { FULL_CONFIG_PATH } = require('../lib/constants');
+
+const cli = new CliInterface({
```
Comment (jondubois) [inline bin/cli.js]: Can we rename this to just `CLI` or `REPLClient` or something else?
CLI stands for 'command line interface' so `CliInterface` is like `Command Line Interface Interface`.

File: bin/cli.js [hunk 2]
```diff
@@ -1,15 +1,297 @@
...
+        passphrase: config.passphrase,
+      });
+
+      cli.options.bindActionArgs = [client];
+    } catch (e) {
+      cli.errorLog("Can't connect to socket");
+    }
+  }
+
+  const ldposAction = async (clientKey, message, action = null) => {
```
Comment (jondubois) [inline bin/cli.js]: Not sure about the name of the last parameter here. If it's the argument for the client, maybe it should be called `arg` or `args` plural? Also what if the method had multiple arguments (not just one)? Should it be an array?

File: package.json [hunk 3]
```diff
@@ -26,9 +26,10 @@
     "mocha": "^8.2.1"
   },
   "dependencies": {
+    "@maartennnn/cli-builder": "^v0.0.7",
```
Comment (jondubois) [inline package.json]: Usually the `v` is omitted. `"^0.0.7",`

**Review by jondubois — CHANGES_REQUESTED** [review]
I made a few comments.

File: bin/cli.js (hunk 2 above)
Comment (maarteNNNN) [reply bin/cli.js:94]: resolved in 8688b5e

File: package.json (hunk 3 above)
Comment (maarteNNNN) [reply package.json]: resolve in 8688b5e

File: bin/cli.js (hunk 1 above)
Comment (maarteNNNN) [reply bin/cli.js]: resolved in 8688b5e

File: bin/cli.js (hunk 2 above)
Comment (maarteNNNN) [reply bin/cli.js]: @jondubois I changed it to `arg`, momentarily it's only used for `sig`, `multisig` and `forging`.

Comment (maarteNNNN) [reply bin/cli.js]: resolved in b2d1cd3

**Review by jondubois — APPROVED** [review]

**Pull request merged**